main.go

```diff
@@ -8,20 +8,22 @@ import (
 )
 
 func main() {
+	var err error
+	tmpl, err = tmpl.ParseFiles("pages/base.gohtml", "pages/home.gohtml")
```

Comment:
```suggestion
	tmpl, err = template.ParseFiles("pages/base.gohtml", "pages/home.gohtml")
```
As `tmpl` is a nil pointer, you must have to create a new template and initialise it, otherwise it will panic

Reply:
good catch! I think I replaced it without noticing after the screencast
Adressed in 5ae4b48